lib/types/meta_data.ex

```diff
@@ -26,7 +26,12 @@ defmodule Kadena.Types.MetaData do
           chain_id: chain_id()
         }
 
-  defstruct [:creation_time, :ttl, :gas_limit, :gas_price, :sender, :chain_id]
+  defstruct creation_time: 0,
+            ttl: 0,
+            gas_limit: 0,
+            gas_price: 0,
+            sender: "",
+            chain_id: %ChainID{id: ""}
```

Comment:
@Odraxs @EdwinGuayacan Why is this required?

Reply:
In the tests we did to the API, we saw that it did not allow null values in the MetaData attributes

Reply:
@EdwinGuayacan ok ok I got you ... let's use 0 as a default value for the ChanID.